### PR TITLE
Enable smoke testing for Dataset Builder snippets.

### LINKED ISSUE #30: Update smoke tests for Dataset Builder snippets.
Right now, these don't run as a standalone python or r script because they are missing the code that Dataset Builder provides for us.

A simple solution would be to prepend the smoke test files with the Python/R code exported to a notebook by Dataset Builder for a particular dataset for smoke testing purposes.

### PR DESCRIPTION
**Pull request by deflaux**
Fixes https://github.com/all-of-us/workbench-snippets/issues/30

### CODE REVIEW

File: snippets/README.md [hunk 1]
```diff
@@ -52,14 +52,26 @@ python2.7 most_recent_measurement_of_interest_test.py
...
 
 ## Integration 'smoke tests'
-The script to auto-generate the Jupyter Snippets Menu configuration also emits both `r_snippets_menu_config_smoke_test.R` and `py_snippets_menu_config_smoke_test.py`. Those scripts each include, respectively, all the R SQL snippets and all the Python SQL snippets. If those scripts are run from the workbench environment and there are no obvious bugs in the snippets, they will run start-to-finish without error. (It won't necessarily catch all bugs, but its a good start.)
+The script to auto-generate the Jupyter Snippets Menu configuration also emits both `r_snippets_menu_config_smoke_test.R` and `py_snippets_menu_config_smoke_test.py`. Those scripts each include, respectively, all the R SQL snippets and all the Python SQL snippets. If those scripts are run from the workbench environment and there are no obvious bugs in the snippets, they will run start-to-finish without error and emit _"Smoke test complete!"_ when they have completed successfully. (This won't necessarily catch all bugs, but its a good start.)
```
Comment (calbach) [inline snippets/README.md]: nit: might be useful to be a bit more prescriptive here, e.g. After opening a notebook in the production workbench environment, upload these smoke test files into Jupyter, then execute the following code from a Python notebook in the same directory.

File: dataset-snippets/README.md [hunk 2]
```diff
@@ -31,8 +31,28 @@ The instructions are identical for both the SQL snippets and the Dataset Builder
...
+## Integration 'smoke tests'
+The script to auto-generate the Jupyter Snippets Menu configuration also emits both `r_dataset_snippets_menu_config_smoke_test.R` and `py_dataset_snippets_menu_config_smoke_test.py`. Those scripts each include, respectively, all the R Dataset Builder snippets and all the Python Dataset Builder snippets. If those scripts are run from the workbench environment and there are no obvious bugs in the snippets, they will run start-to-finish without error and emit _"Smoke test complete!"_ when they have completed successfully. (This won't necessarily catch all bugs, but its a good start.)
+
+The Dataset from Dataset Builder is defined in [r_dataset_snippets_menu_config.smoke_test_setup](./r_dataset_snippets_menu_config.smoke_test_setup) and [py_dataset_snippets_menu_config.smoke_test_setup](./py_dataset_snippets_menu_config.smoke_test_setup), respectively. Update it as needed.
+
+To run the R Dataset Builder snippets smoke tests:
+```
+%%bash
+
+Rscript r_dataset_snippets_menu_config_smoke_test.R  # There will be output, but there should be no errors.
```
Comment (calbach) [inline dataset-snippets/README.md:48]: Do these not also use WORKSPACE_CDR?

**Review by calbach — APPROVED** [review]

File: dataset-snippets/README.md (hunk 2 above)
Comment (deflaux) [reply dataset-snippets/README.md:48]: Not at the moment.
* What Dataset Builder emits has the CDR hard-coded. I pasted that into these `*.smoke_test_setup` files.
* I wanted to minimize local edits, so the only difference between [r_dataset_snippets_menu_config.smoke_test_setup](https://github.com/deflaux/workbench-snippets/blob/add-dataset-to-smoke-test/build/r_dataset_snippets_menu_config.smoke_test_setup#L25) and the code that would be exported to an R notebook by Dataset Builder is just `s/dataset_123456789/YOUR-DATASET-NAME/g`

Please see the comment at the top of [that file]((https://github.com/deflaux/workbench-snippets/blob/add-dataset-to-smoke-test/build/r_dataset_snippets_menu_config.smoke_test_setup#L25)) and let me know if you recommend something different.

File: snippets/README.md (hunk 1 above)
Comment (deflaux) [reply snippets/README.md]: added - thanks!

File: dataset-snippets/README.md (hunk 2 above)
Comment (calbach) [reply dataset-snippets/README.md:48]: Thanks you're right, I got these mixed up - I thought I was commenting on the non-datasets documentation.

**Pull request merged**